src/components/SidePanel/index.tsx

```diff
@@ -30,7 +32,13 @@ export const SidePanelBase = ({
         [styles.isCollapsed]: !isExpanded,
       })}
     >
-      <div className={styles.content}>{children}</div>
+      <div
+        className={makeClassName(styles.content, {
+          [styles.borderlessContent]: borderless,
```

Comment:
This needs a test to make sure a `borderless: true` prop passed to `SidePanel` results in the `styles.borderlessContent` class getting added.